### PR TITLE
Install Static Analysis Tool - "Flow"

### PR DESCRIPTION
**Pull request by cmay20**
**PR Description:**
This branch includes code changes to install the Flow tool. It has successfully been installed and run. Evidence for the testing of Flow can be found in the screenshots below.

---

**Flow Tool Description:** 
[Flow](https://flow.org/) is a static type checker for Javascript. Flow gives you fast feedback while you code by incrementally rechecking your code as you make changes. Flow integrates well with many tools, making it easy to insert into your existing workflow and toolchain.

**Benefits of Flow:**
- Code Faster: Tired of having to run your code to find bugs? Flow identifies problems as you code. Stop wasting your time guessing and checking.
- Code Smarter: It's hard to build smart tools for dynamic languages like JavaScript. Flow understands your code and makes its knowledge available, enabling other smart tools to be built on top of Flow.
- Code Confidently: Making major changes to large codebases can be scary. Flow helps you refactor safely, so you can focus on the changes you want to make, and stop worrying about what you might break.
- Code Bigger: Working in a codebase with lots of developers can make it difficult to keep your master branch working. Flow can help prevent bad rebases. Flow can help protect your carefully designed library from misuse and misinterpretation. And Flow can help you understand the code you wrote six months ago.

**Initial Run/Test (screenshot of output)**

Screenshot 2023-03-17 at 3 52 50 PM

### CODE REVIEW

File: .eslintignore
Comment (liam-gersten) [inline .eslintignore:1]: nice

**Review by liam-gersten — APPROVED** [review]
Tool looks good. If the files change when people run it, maybe we should add them to .gitignore before merging into main.

File: .babelrc
Comment (elzlee) [inline .babelrc:1]: Fantastic

**Review by elzlee — APPROVED** [review]
Looks good

**Review by Ej11 — APPROVED** [review]
Seems good!

**Pull request merged**